### ISSUE TITLE
Scopes no longer working if states are defined as a series of symbols

### ISSUE THREAD

**Issue by TrevorHinesley** [aasm/aasm#365]
In 4.10.0. For instance:

```
aasm column: 'state' do
    state :uploaded, initial: true
    state :tagging_in_progress, :tagging_completed, :approved, :denied

    ... transitions here ...
end
```

Will only have `Model.uploaded` and I believe `Model.tagging_in_progress`, but using `Model.approved` or something doesn't work. Changing to this works:

```
aasm column: 'state' do
    state :uploaded, initial: true
    state :tagging_in_progress 
    state :tagging_completed
    state :approved
    state :denied

    ... transitions here ...
end
```

**Comment by evheny0** [comment]
I have the same issue. As I can see in 
``` ruby
def state_with_scope(name, *args)
  state_without_scope(name, *args)
  create_scope(name) if create_scope?(name)
end
alias_method :state_without_scope, :state
alias_method :state, :state_with_scope
```
the rest of states names, passed to #state method, are just ignored.

**Comment by anilmaurya** [comment]
Fixed with #397
